src/main/java/fr/cnes/sonar/plugins/icode/check/ICodeSensor.java

```diff
@@ -37,15 +39,18 @@
 import org.sonar.api.batch.sensor.issue.NewIssueLocation;
 import org.sonar.api.config.Configuration;
 import org.sonar.api.rule.RuleKey;
+import org.sonar.api.utils.PathUtils;
+import org.sonar.api.utils.System2;
+import org.sonar.api.utils.command.Command;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonar.api.utils.command.Command'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/lequal/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.cnes.sonarqube.plugins%3Asonaricode&pullRequest=31&issues=AWXTgLl8ashydt7kJTGo&open=AWXTgLl8ashydt7kJTGo)

[ISSUE_KEY]: AWXTgLl8ashydt7kJTGo